lib/check-node.js

```diff
@@ -2,11 +2,16 @@ const exec = require('child_process').exec
 const semver = require('semver')
 
 module.exports = function (cb) {
-  exec('node --version', (error, stdout, stderr) => {
-    if (error) return (error)
-    var output =  (stdout + stderr).replace(/(\r\n|\n|\r|\ )/gm,"")
-    var sem = semver.valid(output)
+  try {
+    var sem = process.versions.node
```

Comment:
won't this use the version of node that is bundled with electron? We would want the version of node in the user's PATH, so I'm not sure that this will work for that